atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -81,9 +83,11 @@ def __init__(self, tasker, workflow, labels,
                               "architecture",
                               "vcs-type",
                               "vcs-ref",
-                              "com.redhat.build-host"),
+                              "com.redhat.build-host",
+                              "url"),
                  aliases=None,
-                 dont_overwrite_if_in_dockerfile=("distribution-scope",)):
+                 dont_overwrite_if_in_dockerfile=("distribution-scope",),
+                 info_url_format=None):
```

Comment:
In docstring below, around line 100 (github won't let me create a comment there, as its not part of the diff)